CROSSSECTOR/CPSV-AP/README.md

```diff
@@ -0,0 +1,25 @@
+# dataModel.CPSV-AP
```

Comment:
Why more README?? Possibly these comments should be at notes.yaml

Reply:
README is overwritten so everything here will be lost

Reply:
Deleted